pyproject.toml

```diff
@@ -38,7 +38,7 @@ include = [
 prospector = 'prospector.run:main'
 
 [tool.poetry.dependencies]
-python = ">=3.7,<4.0"
+python = ">=3.7.2,<4.0"
```

Comment:
Is this change necessary for this fix? I imagine it won't affect too much but I'd like to avoid changing python comparability ranges in a bugfix release.

Reply:
Explained in the “Motivation and Context” section of the PR description. Not necessary for the fix itself, but necessary for the tests to not hang indefinitely without the fix. I don't have a strong opinion either way, it feels a bit like a “damned if you do, damned if you don't” kind of situation.

A case could be made that anyone running Python 3.7.0 or 3.7.1 has bigger issues on their hand than losing official support for the latest prospector, and there are also high-profile projects with an official policy of only supporting the latest patch level of each of the respective supported minor branches of Python.

Ultimately, this change was really kinda forced by poetry, because it really *really* didn't want to upgrade to pylint 2.15 without it, but nothing will break in an already working state if I roll it back, so if you prefer me to do that, then can do.

Reply:
Sorry I had not properly read the motivation part. I think you're right that we should update it due to the not being able to have the newer pylint versions.

Prospector really tries to be as vague as possible about version ranges to maximise compatibility with other code linting tools which might be run separately in different contexts, eg IDE plugins which run pylint or whatever. However at some point the legacy code becomes too crufty and leads to further issues. I am maybe just being too paranoid!

Especially since you're right, people stuck on python 3.7.1 then prospector shouldn't hold more up-to-date users to ransom with an older pylint.

I might suggest a `1.9` release instead of a `1.8.5` in this case, what do you think @Pierre-Sassoulas ? I'm easy either way.